### PR TITLE
fix(storage): makes periodsSinceLastPayout 0 if not positive

### PR DESCRIPTION
**Pull request by farinalvaro**
It seems that `lastPayout.getTime()` is 5 minutes ahead of `Date.now()`. This fix defaults to 0 when that time is bigger than now

### CODE REVIEW

**Comment by julianmrodri** [comment]
Just to give some more context we detected they may be micro differences between the `timestamp` in ganache vs the Cache, and that resulted in some negative value during a short period of time. This is a safe check to ensure we handle those micro differences for the UI.

File: src/services/storage/models/agreement.model.ts [hunk 1]
```diff
@@ -57,7 +57,8 @@ export default class Agreement extends Model {
     // Date.now = ms
     // this.lastPayout.getTime = ms
     // this.billingPeriod = seconds ==> * 1000
-    const periods = new BigNumber(Date.now() - this.lastPayout.getTime()).div(this.billingPeriod.times(1000))
+    const timePassedMs = Math.max(Date.now() - this.lastPayout.getTime(), 0)
```
Comment (nduchak) [inline src/services/storage/models/agreement.model.ts:60]: I prefer to separate this:
```
if (Date.now() - this.lastPayout.getTime() < 0) {
  // We ca not fully rely on blockchain time, as it's can be little bit ahead a real time
  return new BigNumber(0)
}
const periods = new BigNumber(Date.now() - this.lastPayout.getTime()).div(this.billingPeriod.times(1000))
return floor
      ? bnFloor(periods)
      : periods

```

Comment (nduchak) [inline src/services/storage/models/agreement.model.ts:60]: I prefer to separate this:
```
if (Date.now() - this.lastPayout.getTime() < 0) {
  // We ca not fully rely on blockchain time, as it's can be little bit ahead a real time
  return new BigNumber(0)
}
const periods = new BigNumber(Date.now() - this.lastPayout.getTime()).div(this.billingPeriod.times(1000))
return floor
      ? bnFloor(periods)
      : periods

```

**Review by nduchak — APPROVED** [review]
LGTM

**Review by julianmrodri — APPROVED** [review]
LGTM

**Review by AuHau — APPROVED** [review]

**Pull request merged**